src/ansys/fluent/core/utils/file_transfer_service.py

```diff
@@ -229,7 +234,12 @@ def upload(
         files = _get_files(file_name, self.host_mount_path)
         if self.client:
             for file in files:
-                if os.path.isfile(file):
+                is_file_exists_on_remote = self.file_exists_on_remote(
+                    os.path.basename(file)
+                )
+                if is_file_exists_on_remote:
+                    print(f"\n{file} already uploaded.\n")
```

Comment:
Shall we use the `warnings` module? 

A more accurate message would say that a file with the *same name* *exists at the remote location*.   We don't know if it's the same file or if it were uploaded.

Reply:
In whole source code we are not using warning module to check the existing files. `(examples.download(), PIMFileTransferService,)` Therefore we are not using it here. We have updated the message now. Thank you.

Reply:
I'm not sure about using print. One thing about using `warnings.warn()` is that allows specific message types to be disabled.

Reply:
Right, I agree. We will update this now. Thank you.

Reply:
Done. Thank you.

Reply:
Overall, as this is a library, we must be careful about writing to stdout.

Reply:
Yes, I agree.